src/Sylius/Bundle/AdminBundle/Resources/views/Customer/Show/_menu.html.twig

```diff
@@ -1,19 +0,0 @@
-<div class="ui fluid labeled four icon item menu">
```

Comment:
Instead of removal, which is forbidden according to our [BC Policy](https://docs.sylius.com/en/latest/book/organization/backward-compatibility-promise.html#templates), we should deprecate this file with the following function:

```twig
{% deprecated "This template is deprecated since Sylius 1.12" %}
```